app/jinja_filters.py

```diff
@@ -534,27 +589,25 @@ def map_summary_item_config(
     ]
 
     if summary_type == "CalculatedSummary":
-        rows.append(
-            SummaryRow(calculated_question, summary_type, False, None, None, None)
-        )
+        rows.append(SummaryRow(calculated_question, summary_type, False, "", "", ""))
 
     return rows
 
 
 @blueprint.app_context_processor
-def map_summary_item_config_processor():
+def map_summary_item_config_processor() -> dict[str, Callable]:
     return dict(map_summary_item_config=map_summary_item_config)
 
 
 @blueprint.app_template_filter()  # type: ignore
 def map_list_collector_config(
-    list_items,
-    icon,
-    edit_link_text=None,
-    edit_link_aria_label=None,
-    remove_link_text=None,
-    remove_link_aria_label=None,
-):
+    list_items: list[dict[str, Union[str, int]]],
+    icon: str,
+    edit_link_text: str = "",
```

Comment:
I think theses should probably be `Optional[str] = None` but will need to find a way to the `.format` method further down.

Reply:
Yeah thats why I didn't make them `Optional[str] = None`.
Ive come up with a solution though and pushed it up